client/core/core_test.go

```diff
@@ -780,6 +783,28 @@ func (rig *testRig) queueConnect(rpcErr *msgjson.Error, matches []*msgjson.Match
 	})
 }
 
+func (rig *testRig) queueCancel(rpcErr *msgjson.Error) {
+	rig.ws.queueResponse(msgjson.CancelRoute, func(msg *msgjson.Message, f msgFunc) error {
```

Comment:
Include `t.Helper()`?

Reply:
I'm not passing `t` to this method. Think I should?

Reply:
Don't think so unless there's a t.Error or t.Fatal anywhere

Reply:
chapp is right.